opendatahub-io/odh-dashboard
# Refactor fetching groups

**Changes proposed in this pull request**
- Add the ability to have `system:authenticated` as a group option
- Refactor log error to add more info

**Test instructions (required)**
- Run a new instance of ODH
- Stop the operator to be able to edit de configmap
- Change the value of `jupyterhub-default-groups-config` Configmap to `admin_groups: 'system:authenticated'`
- Check wether the config options of the dashboard have the settings enable
- Refresh the dashboard and check you can see the settings panel
